showpass/Showpass-E2E-tests
# TA-42 Verify saved events

**Description:**
This spec will be verifying that an event can be saved and found on the ‘Saved Events’ page.
Steps:
- Log into [Create Amazing Experiences](https://beta.showpass.com/) 
- Navigate to the ‘Saved Events’ page and removed all saved items (if any) from there
- Search for and open up an event
- Click the ‘Heart’ icon to save the event
- Add an additional or a couple more events to the ‘Saved Events’ page
- Go to https://beta.showpass.com/account/saved-events/
- Make sure the event shows up on the ‘Saved Events’ page

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-42

**How to test:**
Run the following test suite and ensure it passes
cypress/e2e/user-settings/saved-events.cy.js

**Test execution results:**

<img width="1715" alt="TA-42-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/431c4846-8495-4052-bb05-cdb01b210e38">
